sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/checks/XPathCheckTest.java

```diff
@@ -93,6 +93,13 @@ void test_with_default_namespace() {
     SonarXmlCheckVerifier.verifyIssues("with_default_namespaces.xml", getCheck("//template"));
   }
 
+  @Test
+  void test_with_double_colon() {
+    XPathCheck check = getCheck("//dependency/artifactId[starts-with(text(),'mule-http-connector')]//following-sibling::version[not(starts-with(text(),'1.6'))]");
```

Comment:
I feel like we should add a test with a path that actually contains `namespace-uri` or a single `:` for completeness of the test.

Reply:
pretty good point.